versioned_docs/version-5/teams.mdx

```diff
@@ -132,13 +132,13 @@ A subscription seat is only taken up at the time of accepting the invitation. An
 
 To change a team member’s role, you must have the [Owner role](/#owner-role) or [Admin role](/#admin-role) for that team.
 
-1. Navigate to the team page from the left sidebar of the Studio workspace.
+1. Navigate to the team page from the left sidebar of Stately Studio’s workspace.
 2. Navigate to the **Members** tab on the team page.
-3. Use the **...** triple dot contextual menu alongside the team member’s name in the **Members** list to choose from the options **Make a Viewer**, **Make an Editor**, or **Make an Admin**.
+3. Use the **...** triple dot contextual menu alongside the team member’s name in the **Members** list to choose from the options **Make a Viewer**, **Make an editor**, or **Make an Admin**.
```

Comment:
```suggestion
3. Use the **...** triple dot contextual menu alongside the team member’s name in the **Members** list to choose from the options **Make a Viewer**, **Make an Editor**, or **Make an Admin**.
```
Normally, I'd say we should change the button labels themselves to sentence case except that we're semi-committed to capitalizing these roles for now.